tests/integration/targets/filter_groupby_as_dict/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
 
 - assert:
     that:
-      - result.msg == 'Input is not a sequence'
+      - result.msg is search('Input is not a sequence')
```

Comment:
Are we not supposed to use `==` for strings anymore?

Reply:
No, same as above, it's part of a string now.